x/dsmr/node_test.go

```diff
@@ -29,6 +32,26 @@ var (
 	_ Verifier[tx] = (*failVerifier)(nil)
 )
 
+func newTestingLog() logging.Logger {
+	return logging.NewLogger("dsmr")
+}
```

Comment:
we can replacing this with `logging.NoLog{}`

Reply:
done